frontend/src/pages/acceleratorProfiles/components/TableData.tsx

```diff
@@ -0,0 +1,41 @@
+import { SortableData } from '~/components/table';
```

Comment:
this should not be a `tsx` file. put this in a file named `const.ts`. Also instead of using a components folder at the root.  place all relevant files in their respective screen folders. Unless that component. will be used in both the edit page and list page

```
page/acceleratorProfiles
   - screens
      - list
         - AcceleratorProfiles.tsx
         - AcceleratorProfilesTable.tsx
         - AcceleratorProfilesTableRow.tsx
         - AcceleratorProfileEnableToggle.tsx
         - const.ts
```